src/Contract/Operation/Unfoldable.php

```diff
@@ -18,7 +18,7 @@ interface Unfoldable
      * @psalm-template TKey of array-key
      * @psalm-template T
      *
-     * @psalm-param callable(T...): array<TKey, T> $callback
+     * @psalm-param callable(mixed|T...): (mixed|array<TKey, T>) $callback
```

Comment:
this is based on the Operation annotation: https://github.com/loophp/collection/blob/master/src/Operation/Unfold.php#L38

Reply:
Yep definitely something out of sync that needs to be fixed.